das/daser.go

```diff
@@ -18,21 +19,37 @@ type DASer struct {
 	da   share.Availability
 	hsub header.Subscriber
 
-	cancel context.CancelFunc
-	done   chan struct{}
+	// getter allows the DASer to fetch a header at a certain height
+	// and blocks until it becomes available.
+	getter HeaderGetter
+	// checkpoint from disk -- DASStore stores checkpoint w/ DASCheckpoint key
+	// checkpoint = latest successfully DASed header (reference to it, not the header)
```

Comment:
Can this documentation for what a checkpoint is be placed somewhere in `checkpoint_store.go`? Checkpoint appears to be a height, not a reference.

Reply:
yes

Reply:
done